unit_tests/test_table.py

```diff
@@ -1472,9 +1472,18 @@ def read_row(self, *args, **kwargs):
         self.read_row_calls.append((args, kwargs))
         return self.read_row_result
 
-    def read_rows(self, *args, **kwargs):
+    def yield_rows(self, *args, **kwargs):
         self.read_rows_calls.append((args, kwargs))
-        return self.read_rows_result
+        self.read_rows_result.consume_all()
+        rows_dict = self.read_rows_result.rows
+        while True:
+            try:
+                for row_key in sorted(rows_dict):
+                    curr_row_data = rows_dict.pop(row_key)
+                    yield curr_row_data
+                self.read_rows_result.consume_next()
```

Comment:
`consume_next()` after `consume_all()` doesn't seem like it should work here.

Reply:
@tseaver In unit test consume_next() and consume_all() are only used to increment counters. They are not used to populate rows as in actual bigtable. And this counter is used to in AssertEqual. In unit test  consume_next() and consume_all() don't do anything besides incrementing counter. I am just calling it for maintaining counter value to avoid counter assert errors.

Reply:
please figure out how to remove the `consume_next` and `consume_all` methods